rosbag2_storage_sqlite3/src/rosbag2_storage_sqlite3/sqlite_storage.cpp

```diff
@@ -320,17 +320,18 @@ void SqliteStorage::write(
   commit_transaction();
 }
 
-void SqliteStorage::set_read_order(const rosbag2_storage::ReadOrder & read_order)
+bool SqliteStorage::set_read_order(const rosbag2_storage::ReadOrder & read_order)
 {
   if (read_order.sort_by == rosbag2_storage::ReadOrder::PublishedTimestamp) {
-    throw std::runtime_error("Not Implemented - PublishedTimestamp read order.");
```

Comment:
Maybe worth logging warnings or at least debug?

Reply:
log debug? I'm pretty up on log statements in general - in my experience from having to debug complex (dozens of processes) robot applications on devices running in the field, detailed text logs are often the only way to even get a sense of what happened on somebody else's thing after the fact, especially if it's not 100% reproducible. That's a much deeper general software engineering philosophy and my preference is based on my specific experiences/projects, but regardless, a debug log doesn't hurt especially if it uses macros that can be compiled out.